ansible/files/postgresql_config/postgresql.conf.j2

```diff
@@ -428,39 +428,7 @@ effective_cache_size = 128MB
 # REPORTING AND LOGGING
 #------------------------------------------------------------------------------
 
-# - Where to Log -
-
-log_destination = 'csvlog'		# Valid values are combinations of
-					# stderr, csvlog, syslog, and eventlog,
-					# depending on platform.  csvlog
-					# requires logging_collector to be on.
-
-# This is used when logging to stderr:
-logging_collector = on		# Enable capturing of stderr and csvlog
-					# into log files. Required to be on for
-					# csvlogs.
-					# (change requires restart)
-
-# These are only used if logging_collector is on:
-log_directory = '/var/log/postgresql'			# directory where log files are written,
-					# can be absolute or relative to PGDATA
-log_filename = 'postgresql.log'	# log file name pattern,
-					# can include strftime() escapes
-log_file_mode = 0640			# creation mode for log files,
-					# begin with 0 to use octal notation
-log_rotation_age = 0			# Automatic rotation of logfiles will
-					# happen after that time.  0 disables.
-log_rotation_size = 0		# Automatic rotation of logfiles will
-					# happen after that much log output.
-					# 0 disables.
-#log_truncate_on_rotation = off		# If on, an existing log file with the
-					# same name as the new log file will be
-					# truncated rather than appended to.
-					# But such truncation only occurs on
-					# time-driven rotation, not on restarts
-					# or size-driven rotation.  Default is
-					# off, meaning append to existing files
-					# in all cases.
+include = '/etc/postgresql/logging.conf'
```

Comment:
nit: Can we store it with the rest in `/etc/postgresql-custom` dir?

Reply:
The custom dir is currently used for config that can be dynamically updated (whether by user, or internal tooling). Going to leave as-is unless there's a strong objection.